src/tools/dev/scripts/bv_support/bv_tbb.sh

```diff
@@ -79,9 +79,8 @@ function bv_tbb_host_profile
         #   TBB related projects. I want to keep it set here since I am afraid of breaking other packages
         #   other than ospray
         if [[ "$USE_SYSTEM_TBB" == "no" ]]; then
-            echo "SETUP_APP_VERSION(TBB ${TBB_VERSION})" >> $HOSTCONF
-            echo "VISIT_OPTION_DEFAULT(TBB_ROOT \${VISITHOME}/tbb/\${TBB_VERSION}/\${VISITARCH})" >> $HOSTCONF
-            echo "VISIT_OPTION_DEFAULT(VISIT_TBB_DIR \${VISITHOME}/tbb/\${TBB_VERSION}/\${VISITARCH})" >> $HOSTCONF
+            echo "VISIT_OPTION_DEFAULT(TBB_ROOT \${VISITHOME}/tbb/${TBB_VERSION}/\${VISITARCH})" >> $HOSTCONF
```

Comment:
Shouldn't there be a

echo "set(TBB_VERSION ${TBB_VERSION})" >> $HOSTCONF

added here?

Reply:
There isn't any place within visit that uses TBB_VERSION, so I though, why set it?

I kep the current config-sites  setting TBB_VERSION just to save myself having to type the version in the VISIT_TBB_DIR and TBB_ROOT paths.

Reply:
Got it. That makes sense.